public/app/features/inspector/QueryInspector.tsx

```diff
@@ -19,6 +20,7 @@ interface ExecutedQueryInfo {
 }
 
 interface Props {
+  instanceId?: string; // Must match the prefix of the requestId of the query being inspected. For updating only one instance of the inspector in case of multiple instances, ie Explore split view
```

Comment:
I was really torn with the name on this. Ideally it should include `requestId` but opted for something more indicating it's use, since most places won't use it. Let me know if you disagree